cargo-shuttle/src/client.rs

```diff
@@ -24,7 +25,7 @@ pub struct ShuttleApiClient {
     api_url: String,
     api_key: Option<ApiKey>,
     /// alter behaviour to interact with the new platform
-    beta: bool,
+    pub beta: bool,
```

Comment:
nit: it shouldn't need to be `pub` ?

Reply:
I needed it to be pub for a check outside the client. This comment is related to this one too: https://github.com/shuttle-hq/shuttle/pull/1734#discussion_r1565290876, where I guess what we'll do is have two different functions. I'll update it based on this remark.